packages/migrations/test/utils/testkit.ts

```diff
@@ -113,7 +113,9 @@ export async function initMigrationTestingEnvironment() {
       await runPGMigrations({ slonik });
     },
     async done(deleteDb = true) {
-      deleteDb ?? (await db.query(`DROP DATABASE ${dbName};`));
+      if (deleteDb) {
+        await db.query(`DROP DATABASE ${dbName};`);
+      }
```

Comment:
@n1ru4l  I think the previous condition was incorrect

when `deleteDb: true` => do nothing
when `deleteDb: false` => do nothing
when `deleteDb: null | undefined` => delete db